kodilla-intro/src/main/java/Grades.java

```diff
@@ -1,2 +1,35 @@
 public class Grades {
+    private int[] grades;
+    private int size;
+    private int lastGrade;
+
+    public Grades() {
+        this.grades = new int[10];
+        this.size = 0;
+    }
+
+    public void add(int value) {
+        if (this.size == 10) {
+            return;
+        }
+        this.grades[this.size] = value;
+        this.size++;
+
+        int sum = 0;
+        for( int i = 0; i <= 10;i++) {
+            sum += grades[ i ];
+
+
+
+            int numberOfElements = grades.length;
+            int srednia = sum/numberOfElements;
+
+
+
+
+            System.out.println("Sum of array elements is:"+sum);
+            System.out.println("srednia"+ srednia);
+        }
```

Comment:
co ta cała metoda add robi? Ona się powinna skończyć w lini 16. Potem widze, ze liczysz srednia - ale to powinno sie odbyc w nowej metodzie.